coalib/results/Diff.py

```diff
@@ -278,18 +278,37 @@ def modified(self):
         return result
 
     @property
+    def modified(self):
+        """
+        Calculates the modified file, after applying the Diff to the original.
+
+        This property also adds linebreaks at the end of each line.
+        If no newline was present at the end of file before, this state will
+        be preserved, except if the last line is deleted.
+        """
+        return self._generate_linebreaks(self._raw_modified())
+
+    @property
     def unified_diff(self):
         """
         Generates a unified diff corresponding to this patch.
 
+        Each change will be displayed on its own line. Additionally, the
+        unified_diff preserves the EOF-state of the original file. This
+        means that the Diff will only have a linebreak on the last line,
```

Comment:
Diff in double backticks so it gets rendered as code ;) Usually code references like these should be enclosed in double-backticks so people know that this is a legal identifier :+1: